internal/pkg/deploy/cloudformation/stack.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package cloudformation
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/cloudformation"
+)
+
+// DeleteStack calls the CloudFormation DeleteStack API with the input stack name.
+func (cf CloudFormation) DeleteStack(stackName string) error {
```

Comment:
Seems like these 2 functions, `DeleteStack` and `WaitForStackDelete` are only used by the `DeleteStackAndWait`. Should we make these 2 functions private? I guess they are useful in case someone does not want to wait for stack deletion.

Reply:
I don't see any reason these shouldn't be part of the public API, so I left them exported in case someone else wants to use them.